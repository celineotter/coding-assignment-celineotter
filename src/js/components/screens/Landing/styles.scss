@import 'src/styles/base.scss';

.wrapper {
  text-align: center;
}

.logo {
  height: 80px;
}

.header {
  background-color: $black;
  color: $white;
  height: 150px;
  padding: 20px;
}

.title {
  font-size: 1.5em;
}

.intro {
  font-size: large;
}
