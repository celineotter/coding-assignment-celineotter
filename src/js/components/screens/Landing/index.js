// @flow

import React from 'react';

import './styles.css';

const Landing = () => {
  return (
    <div styleName="wrapper">
      <header styleName="header">
        <h1 styleName="title">Ancestry Coding Challenge</h1>
      </header>
      <p styleName="intro" testid="intro">
        My react app!
      </p>
    </div>
  );
};

export default Landing;
