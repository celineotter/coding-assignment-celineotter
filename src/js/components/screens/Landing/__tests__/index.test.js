import React from 'react';
import { shallow } from 'enzyme';

import Landing from '@screens/Landing';

it('renders landing text', () => {
  const landing = shallow(<Landing />);

  expect(landing).toIncludeText('Ancestry Coding Challenge');
  expect(landing.find({ testid: 'intro' })).toHaveText('My react app!');
  expect(landing).toMatchSnapshot();
});
