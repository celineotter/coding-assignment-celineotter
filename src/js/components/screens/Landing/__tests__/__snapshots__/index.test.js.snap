// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders landing text 1`] = `
<div
  className="src-js-components-screens-Landing-___styles__wrapper___1ia3v"
>
  <header
    className="src-js-components-screens-Landing-___styles__header___2IFnP"
  >
    <h1
      className="src-js-components-screens-Landing-___styles__title___1GTe4"
    >
      Ancestry Coding Challenge
    </h1>
  </header>
  <p
    className="src-js-components-screens-Landing-___styles__intro___2mDAK"
    testid="intro"
  >
    My react app!
  </p>
</div>
`;
